drawBot/drawBotDrawingTools.py

```diff
@@ -406,8 +406,14 @@ def saveImage(self, paths, multipage=None):
             context = getContextForFileExt(ext)
             if context is None:
                 raise DrawBotError("Could not find a supported context for: '%s'" % ext)
+            allowedSaveImageOptions = set(optionName for optionName, optionDoc in context.saveImageOptions)
```

Comment:
this has to be tested outside the loop as this is possible:
multiple files and each with their own options... 
`saveImage([path.png, path.pdf, path.mp4], imagePNGGamma=5, ffmpegCodec="mpeg4")`